Sprint-2/2-mandatory-debug/1.js

```diff
@@ -1,13 +1,21 @@
 // Predict and explain first...
 //  =============> write your prediction here
+    // The function returns nothing
+//function sum(a, b) {
+  //return;
+  //a + b;
```

Comment:
Whatever code you write after return statement become useless because  javascript finish  complying your function after return statement